templates/cassiopeia/index.php

```diff
@@ -41,9 +41,14 @@
 // Load custom Javascript file
 HTMLHelper::_('script', 'user.js', ['version' => 'auto', 'relative' => true]);
 
+//HTMLHelper::_('script', 'media/vendor/responsifyjs/responsify.js', ['relative' => false]);
+
```

Comment:
Templates don't need to be aware of scripts needs by a plugin

Reply:
@proman24 can you remove this unnecessary lines